app/assets/javascripts/teams.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function() {
+    $('#team-sources input[id^=team_sources_attributes]:radio').change(function() {
```

Comment:
You could use a data-attribute instead of the ID. From my experience, this usually makes the intention clearer (especially given the fact that we don't test JS code 🙈 ). So maybe: `data-behavior="source-switcher"` or something like this.